<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTemplatesForm</class>
 <widget class="QWidget" name="CreateTemplatesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
              }
QTabWidget {
               color: blue;
               padding: 4px;
               border-radius: 5px;
               border: 1px solid rgb(63, 127, 191); 
               border-style: outset;
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QWidget {
               color: rgb(32, 74, 135);
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QScrollBar {
			   background-color: rgb(240, 240, 240);
              }
QLabel {
               color: rgb(32, 74, 135);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QLineEdit {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }
QComboBox {
			   color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
               font: 9pt &quot;MS Shell Dlg 2&quot;;
			   background-color: rgb(255, 255, 255);
              }
QTextEdit {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }
QSpinBox {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }
QListView {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel {
               color: darkblue;
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
	           font: 12pt &quot;MS Shell Dlg 2&quot;;
              }</string>
     </property>
     <property name="text">
      <string>CREATE TEMPLATE</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>300</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>798</width>
           <height>298</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Library name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="libraryNameline"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Function</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="functionLine"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Table name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="tableNameLine"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Table column</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="tableColumnLine"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Column type</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="columnTypeLine"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QTextEdit" name="descriptionEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="backButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
                color: darkblue;
               padding: 4px;
               border: 1px solid rgb(0, 85, 255); 
               border-radius: 5px;
               border-style: outset;
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(0, 78, 235);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="submitButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
               color: white;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
               max-width: 120px;
               background-color: rgb(0, 85, 255);
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: darkblue;
               border: 1px solid grey;
               background-color: rgb(255, 255, 255);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
			    background-color: rgb(0, 78, 235);
               }</string>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
